Jumpscale/clients/blockchain/tfchain/tests/200_reservation.py

```diff
@@ -0,0 +1,82 @@
+    c._explorer_get = explorer_client.explorer_get
+    c._explorer_post = explorer_client.explorer_post
+
+    # the devnet genesis seed is the seed of the wallet,
+    # which receives all block stakes and coins in the genesis block of the tfchain devnet
+    DEVNET_GENESIS_SEED = "image orchard airport business cost work mountain obscure flee alpha alert salmon damage engage trumpet route marble subway immune short tide young cycle attract"
+
+    # create a new devnet wallet
+    w = c.wallets.new("mywallet", seed=DEVNET_GENESIS_SEED)
+    # we create a new wallet using an existing seed,
+    # such that our seed is used and not a new randomly generated seed
+
+    # a tfchain (JS) wallet uses the underlying tfchain client for all its
+    # interaction with the tfchain network
+    assert w.network_type == "DEV"
+
+    w.capacity._notary_client_ = NotaryClientStub(c)
+
+    # create user threebot record
+    explorer_client.threebot_record_add(ThreeBotRecord(
+        identifier=1,
+        names=[BotName(value="user3bot")],
+        addresses=[],
+        public_key=w.key_pair_get(w.address).public_key,
+        expiration=1552581420,
+    ))
+    # create grid broker threebot record
+    explorer_client.threebot_record_add(ThreeBotRecord(
+        identifier=2,
+        names=[BotName(value="development.broker")],
+        addresses=[],
+        public_key=PublicKey.from_json("ed25519:e4f55bc46b5feb37c03a0faa2d624a9ee1d0deb5059aaa9625d8b4f60f29bcab"),
+        expiration=1552581420,
+    ))
+
+    # try to reserve a 0-os VM
+    result = w.capacity.reserve_zos_vm("user@mail.com", "user3bot")
+    assert result.submitted
+
+    reservation = w.capacity._notary_client.get(result.transaction.data.value.decode())
+    assert reservation
```

Comment:
don't you want to validate that the reservation properties are defined as expected?